packages/core/src/browser/shared-style.ts

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (C) 2019 TypeFox and others.
+ * Copyright (C) 2019 Typefox and others.
```

Comment:
it is indeed `TypeFox` :-)

Reply:
Oops sorry about that! I must have mistyped when changing the author from Ericsson to the original author TypeFox when refactoring.